src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -12,19 +14,66 @@
 
 public class Operations {
 
-    public static <T extends IPerson> Set<T> getByFriendliness(Configuration<T> configuration, double value) {
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} whose friendliness value is equal to the provided value.
+     *
+     * @param configuration The {@link Configuration} to mutate.
+     * @param value         The {@link double} all {@link IPerson}s should have.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
+    public static <T extends IPerson> Set<T> getByExactValue(final Configuration<T> configuration, final double value) {
         return filterMatchingPersons(configuration, t -> t.getValue() == value);
     }
 
-    public static <T extends IPerson> Set<T> getByCloseToAssociation(Configuration<T> configuration, Association association) {
-        return filterMatchingPersons(configuration, t -> Association.closestToValue(t.getValue()) == association);
+
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} whose friendliness value fits the provided association.
+     *
+     *
+     * @param configuration The {@link Configuration} to mutate.
+     * @param association   The {@link Association} all {@link IPerson}s should have.
+     * @param <T>           The type of {@link IPerson} this configuration holds.
+     * @return              The mutated immutable {@link Configuration}.
+     */
+    public static <T extends IPerson> Set<T> getByAssociation(final Configuration<T> configuration, final Association association) {
+        if (association == Association.ALLY) return getFriends(configuration);
+        if (association == Association.ENEMY) return getEnemies(configuration);
+        return getByExactValue(configuration, association.getValue());
     }
 
-    public static <T extends IPerson> Set<T> getByAssociation(Configuration<T> configuration, Association association) {
-        double value = association.getValue();
-        return getByFriendliness(configuration, value);
+
+    /**
+     * Returns an immutable {@link Set} of all persons in the provided {@link Configuration} that are marked as friends.
```

Comment:
They're not *marked* as friends. Describe this like it really is: their friendliness value is higher than `0` (which is equal to `Standard#STANDARD_NEUTRAL`)